plugins/org.jboss.tools.hibernate.runtime.v_5_2/.classpath

```diff
@@ -11,7 +11,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/hibernate-commons-annotations-5.0.1.Final.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/hibernate-core-5.2.11.Final.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/hibernate-jpa-2.1-api-1.0.0.Final.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/hibernate-tools-5.2.6.Final.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/hibernate-tools-5.2.6.Final.jar" sourcepath="/Users/koen/.m2/repository/org/hibernate/hibernate-tools/5.2.6.Final/hibernate-tools-5.2.6.Final-sources.jar"/>
```

Comment:
@koentsje i think this line should be fixed

Reply:
@bdshadow You are very right! Thx.